packages/sdk/src/base/directus.ts

```diff
@@ -159,7 +156,18 @@ export class Directus<T extends TypeMap> implements IDirectus<T> {
 		);
 	}
 
+	collections<C extends string, I = TypeOf<T, C>>(collection: C): CollectionsHandler<I> {
+		return (
+			this._collections[collection] ||
+			(this._collections[collection] = new CollectionsHandler<T>(collection, this.transport))
+		);
+	}
+
 	items<C extends string, I = TypeOf<T, C>>(collection: C): IItems<I> {
 		return this._items[collection] || (this._items[collection] = new ItemsHandler<T>(collection, this.transport));
 	}
+
+	settings<C extends string, I = TypeOf<T, C>>(collection: string): SettingsHandler<I> {
+		return this._settings[collection] || (this._settings[collection] = new SettingsHandler<T>(this.transport));
+	}
```

Comment:
What's the requirement on `collection` under settings? Settings aren't scoped to collections 🤔